### PR TITLE
 Created TextArea and Button components

### PR DESCRIPTION
**Pull request by XkryGit**
Issue "#24 Create a wrapper for textarea and button" task done.

### CODE REVIEW

File: apps/react-app/src/common/forms/mf-button-styles.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { css } from '@emotion/css';
+import { ColorPrincipal, ColorSecondary } from './mf-color.constants';
+
+export const buttonStyle = css`
+  background-color: ${ColorPrincipal};
+  border: none;
+  padding: 15px;
+  border-radius: 8px;
+  font-family: Avenir, sans-serif;
```
Comment (brauliodiez) [inline apps/react-app/src/common/forms/mf-button-styles.ts:9]: We could think about creating some global CSS styles or theming to hold settings like this

Adding @nasdan and @manudous to the loop

File: apps/react-app/src/common/forms/mf-button-component.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+import * as classes from './mf-button-styles';
+
+type Props = React.ButtonHTMLAttributes<HTMLButtonElement>;
+
+export const ButtonComponent: React.FC<Props> = props => {
```
Comment (brauliodiez) [inline apps/react-app/src/common/forms/mf-button-component.tsx:7]: Naming is always hard, and there can be several valid options.

I think could use the following

MfButton

File: apps/react-app/src/common/forms/mf-text-area.component.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+import * as classes from './mf-text-area-styles';
+
+type Props = React.TextareaHTMLAttributes<HTMLTextAreaElement>;
+
+export const TextAreaComponent: React.FC<Props> = props => {
```
Comment (brauliodiez) [inline apps/react-app/src/common/forms/mf-text-area.component.tsx:7]: We could name this

MfTextArea

File: apps/react-app/src/common/forms/mf-button-component.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (brauliodiez) [inline apps/react-app/src/common/forms/mf-button-component.tsx:1]: Add an index.ts under forms folder, something like

_/apps/react-app/src/common/forms/index.ts_

```ts
export * from './mf-button.component.tsx';
export * from './mf-text-area.component.tsx';

```

File: apps/react-app/src/common/forms/mf-button-component.tsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+import { cx } from '@emotion/css';
```
Comment (brauliodiez) [inline apps/react-app/src/common/forms/mf-button-component.tsx:2]: Filenaming

right now you have

_mf-button-component.tsx_

We use dash for the domain, and . to indicate the type (is just a rule that we use, not a universal standard).

Filenames should be:

mf-button.component.tsx

mf-button.component.styles.tsx

mf-text-area.component.tsx

mf-text-area.component.styles.ts

File: package.json [hunk 6]
```diff
@@ -35,5 +35,8 @@
   },
   "engines": {
     "node": ">=16.0.0"
+  },
+  "dependencies": {
+    "@emotion/css": "^11.10.6"
```
Comment (manudous) [inline package.json:40]: Remove this dependency from the general package.json and install @emotion/css only in react project.

Comment (XkryGit) [reply package.json:40]: Done.

File: apps/react-app/src/common/forms/mf-button-styles.ts (hunk 1 above)
Comment (XkryGit) [reply apps/react-app/src/common/forms/mf-button-styles.ts:9]: Done.

File: apps/react-app/src/common/forms/mf-button-component.tsx (hunk 2 above)
Comment (XkryGit) [reply apps/react-app/src/common/forms/mf-button-component.tsx:7]: Done.

File: apps/react-app/src/common/forms/mf-text-area.component.tsx (hunk 3 above)
Comment (XkryGit) [reply apps/react-app/src/common/forms/mf-text-area.component.tsx:7]: Done.

File: apps/react-app/src/common/forms/mf-button-component.tsx (hunk 4 above)
Comment (XkryGit) [reply apps/react-app/src/common/forms/mf-button-component.tsx:1]: Done.

File: apps/react-app/src/common/forms/mf-button-component.tsx (hunk 5 above)
Comment (XkryGit) [reply apps/react-app/src/common/forms/mf-button-component.tsx:2]: Done.

**Pull request merged**